src/features/feed/components/Feed.tsx

```diff
@@ -32,9 +32,8 @@ function Feed() {
         />
         <Box className="feed-posts" sx={{ mb: "100px" }}>
           {feed.map((post) => (
-            <Box sx={{ pt: "30px" }}>
+            <Box key={post.id} sx={{ pt: "30px" }}>
```

Comment:
Assigned key to direct child component rather than a nested one. Fixed the React warning about child components needing unique keys.